log-reader.tf

```
@@ -28,7 +28,7 @@ data "aws_iam_policy_document" "log_reader" {
     ]
 
     resources = [
-      "${aws_cloudwatch_log_group.this.arn}:log-stream:*"
+      "arn:aws:logs:${data.aws_region.this.name}:${data.aws_caller_identity.current.account_id}:log-group:*:log-stream:*"
```

This would allow the log reader to read any logs in the account.
It would be best to limit this down to a set of log groups for this beanstalk app.

The following would do this; however, you may want to create a local variable for the log group prefix and reuse it in `outputs.tf` (`log_group_name`).

```suggestion
      "arn:aws:logs:${data.aws_region.this.name}:${data.aws_caller_identity.current.account_id}:log-group:/aws/elasticbeanstalk/${aws_elastic_beanstalk_application.this.name}/*:log-stream:*"
```

---

Yeah, great suggestion! Thank you.